liquibase-standard/src/main/java/liquibase/sqlgenerator/core/AddPrimaryKeyGenerator.java

```diff
@@ -37,7 +37,7 @@ public ValidationErrors validate(AddPrimaryKeyStatement addPrimaryKeyStatement,
             }
         }
 
-        if (!((database instanceof OracleDatabase) || (database instanceof AbstractDb2Database))) {
+        if (!((database instanceof OracleDatabase) || (database instanceof AbstractDb2Database) || (database instanceof PostgresDatabase))) {
```

Comment:
Please update PR title to reflect the changes to Postgresql too.

Reply:
Done